bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
making sure the list of things you need to remove is complete is a lot harder imho...

it should be possible to determine which variables are being checked in a Python script by checking for `os.environ`, `os.getenv` usage?